django/transformer/transform/dataframe.py

```diff
@@ -83,7 +80,16 @@ def merge_by_attributes(data, attributes: List[Attribute], primary_key_value: st
     """
     merged_data = defaultdict(list)
     for attribute in attributes:
-        data_for_attribute = {key: value for key, value in data.items() if key[0] == attribute.path}
+        data_for_attribute = {
+            key: value
+            for key, value in data.items()
+            if key[0] == attribute.path
+            or (
+                key[0] == CONDITION_FLAG
+                and key[1]
+                in (c.sql_column.table_name_with_joins() for g in attribute.input_groups for c in g.conditions)
+            )
```

Comment:
the function documentation needs to be updated with the new format (use table_name_with_joins instead of (table, column)). if you feel like documenting this not-so-easy condition, feel free (I could not find  away to formulate it clearly haha) ;)